.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+optuna.integration:
+  - optuna/integration/**/*
+
+optuna.pruners:
+  - optuna/pruners/**/*
+
+optuna.samplers:
+  - optuna/samplers/**/*
+
+optuna.storages:
+  - optuna/storages/**/*
+
+optuna.testing:
+  - optuna/testing/**/*
+
+optuna.visualization:
+  - optuna/visualization/**/*
+
+optuna.study:
+  - optuna/study.py
+
+optuna.trial:
+  - optuna/trial.py
```

Comment:
How about adding `optuna.importance` label for the hyperparameter importance feature introduced by #946?

Reply:
Thanks for your inputs. I considered that as well but omitted it for now as its experimental, but it's not a strong opinion, let me add it.

Reply:
Added!